components/bl-autocomplete-component/src/index.js

```diff
@@ -21,10 +21,10 @@ export default function AutocompleteComponent({ component, eventHandlers }) {
     setOptionsList(validate(options));
   }, [options]);
 
+  const hasGroup = optionsList[0]?.hasOwnProperty("groupLabel");
```

Comment:
do we really need the `hasOwnProperty ` here?

Reply:
How else can we check the object for the presence of the "groupLabel" property?

Reply:
`optionsList[0]?.groupLabel`?